app/src/main/java/com/example/keniphhomemaintenance/maintenance_screen/MaintenanceItem.kt

```diff
@@ -0,0 +1,8 @@
+package com.example.keniphhomemaintenance.maintenance_screen
+
+data class MaintenanceItem(
+    val icon: Int,
+    val title: String,
+    val dueDate: String,
```

Comment:
dueDate is a string for now. Later will be a Date object of some kind. User will select a date from a popup calendar or something. Used string for current simplicity